src/components/InstitutionListFilters/index.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import { userEvent, within } from "@storybook/testing-library";
+import { fn } from "@storybook/test";
+import { SearchParamsProvider } from "../Contexts/SearchParamsContext";
+import InstitutionListFilters from "./index";
+
+const meta: Meta<typeof InstitutionListFilters> = {
+  title: "Admin Portal / InstitutionListFilters",
+  component: InstitutionListFilters,
+  decorators: [
+    (Story) => (
+      <SearchParamsProvider>
+        <Story />
+      </SearchParamsProvider>
+    ),
+  ],
+  args: {
+    onChange: () => fn(),
```

Comment:
I'm not super familiar with the mock function generator yet, but it seems this is not triggering anything in the "Actions" tab ATM.

If you remove the anonymous function portion of this, it seems to work (aka, just call `fn()`).

```suggestion
    onChange: fn(),
```

Reply:
Thanks, that fixed it! [9db93b5](https://github.com/CBIIT/crdc-datahub-ui/pull/669/commits/9db93b5d736dfe54784637d398c2aa02696e400a)